spec/features/post_message_spec.rb

```diff
@@ -1,9 +1,9 @@
-# feature "post message" do 
-#   scenario 'users can post message to chitter ' do 
-#     visit('/messages/post')
-#     fill_in('message', with: 'I have posted a new message!')
-#     click_button('POST')
+feature "post message" do 
+  scenario 'users can post message to chitter ' do 
+    visit('/messages/post')
+    fill_in('message', with: 'I have posted a new message!')
```

Comment:
I love the clarity of the messages for testing 'I have posted a new message' tells me the intent of the test, and what to expect